go.mod

```diff
@@ -16,6 +16,7 @@ require (
 	github.com/schollz/progressbar/v3 v3.8.3
 	github.com/stretchr/testify v1.7.3
 	github.com/turbolent/prettier v0.0.0-20220320183459-661cc755135d
+	github.com/ElrondNetwork/big-int-util v0.1.0
```

Comment:
This library is GPL licensed, so unfortunately cannot be used.

Reply:
NOOOO

Reply:
that's upsetting haha, I did notice the GPL license yesterday but a quick search told me that Apache and GPL were compatible. I guess they work together if you reverse the dependency order (GPL project depends on Apache) since Apache is less restrictive. My bad.